.gitignore

```diff
@@ -1,3 +1,5 @@
 /vendor
 /examples/vendor
-/.php_cs.cache
\ No newline at end of file
+/.php_cs.cache
+/build/logs/clover.xml
+examples/composer.lock
```

Comment:
- `composer.lock` in the root of the library should be removed from the repository
- `examples/composer.lock` should be added to the repository